tests/test_prettytable.py

```diff
@@ -416,6 +447,31 @@ def city_data_from_csv() -> PrettyTable:
     return from_csv(csv_fp)
 
 
+@pytest.fixture(scope="function")
+def city_data_from_mediawiki() -> PrettyTable:
```

Comment:
This function isn't called anywhere, and therefore `from_mediawiki` isn't tested. Please can you check it?